db/migrate/20230221194145_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :Title
+      t.text :Text
```

Comment:
- [ ] Please make sure that the name of the columns is not capitalized in order to differentiate between the model's columns and the model's class name itself. `title` and `text`

_Kindly apply the changes in all similar situations_